app/react/components/datatables/TableSettingsMenuAutoRefresh.tsx

```diff
@@ -44,14 +45,9 @@ export function TableSettingsMenuAutoRefresh({ onChange, value }: Props) {
               isCheckVisible ? styles.alertVisible : styles.alertHidden,
               styles.check
             )}
-            onTransitionEnd={() => setIsCheckVisible(false)}
+            // onTransitionEnd={() => setIsCheckVisible(false)}
```

Comment:
why is this hidden?

Reply:
thank-you for finding this! - un-commented